lighthouse-core/audits/accessibility/empty-table-header.js

```diff
@@ -34,6 +34,7 @@ class EmptyTableHeader extends AxeAudit {
       title: str_(UIStrings.title),
       failureTitle: str_(UIStrings.failureTitle),
       description: str_(UIStrings.description),
+      scoreDisplayMode: AxeAudit.SCORING_MODES.INFORMATIVE,
```

Comment:
`empty-table-header` only ever [passes or is `incomplete`](https://github.com/dequelabs/axe-core/commit/813ee7efdf77a55a1c09a3b8836007917ead91b5#diff-b61c1b80dffa3a75fab95a56f0731b63386f236ba1adc6d7ef2bf4c2a1420d53R5). Based on #10072, we should be handling those audits as `scoreDisplayMode: 'informative'`, so here we are.

I was out on leave for that PR so I don't have all the background, but today it does seems a little strange to not have it count for anything when passing, even if `axe` can't be sure the failing case is really failing.

That said, it seems like our `a11y_tester` example really is a failure, and it's not clear to me why "Unable to determine if element has children" is the best we can do (the `<th>` is plainly empty) and if it's useful to show that to users.

Maybe for now we should drop this audit after all and re-add it when/if it gains some certainty?

Reply:
update:

https://github.com/dequelabs/axe-core/issues/2956

thx @straker and @padmavemulapati :)